community/index/src/main/java/org/neo4j/index/gbptree/GBPTree.java

```diff
@@ -447,25 +441,31 @@ public void merge( KEY key, VALUE value, ValueMerger<VALUE> valueMerger ) throws
         {
             goToRoot( cursor );
 
-            SplitResult<KEY> split = treeLogic.insert( cursor, key, value, valueMerger, options,
+            treeLogic.insert( cursor, structurePropagation, key, value, valueMerger, options,
```

Comment:
That is a concern we had when making this change as well, perhaps it makes sense to add